src/status_im/chat/handlers/receive_message.cljs

```diff
@@ -64,10 +66,10 @@
   (let [ct       (if (= content-type c/content-type-command)
                    c/content-type-wallet-command
                    c/content-type-wallet-request)
-        message' (assoc message :clock-value 0
-                                :message-id (random/id)
-                                :chat-id wallet-chat-id
-                                :content-type ct)]
+        message' (dissoc (assoc message :clock-value 0
```

Comment:
maybe better to use `->` here

Reply:
A good point, it does look better. In fact, it should be a function. That way we can unit test it, too.